### DISCUSSION TITLE
[Bug] No Data in UI when time range is bigger than one hour

### DISCUSSION THREAD

**Discussion by marco-simoncini** [apache/skywalking#8254] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-rocketbot-ui)

### What happened

I'm running skywalking 8.8.1 using docker compose as described in https://github.com/apache/skywalking/blob/master/docker . I'm using postgres 11.3 as backend storage.  

After selecting a time range bigger than one hour every diagrams get null values.

Example: 
time range 58 min 
![immagine](https://user-images.githubusercontent.com/70517698/144879286-87755842-6267-4d8c-a910-342354f4aeb0.png)

time range 1h 18min 
![immagine](https://user-images.githubusercontent.com/70517698/144882846-98dfed65-77f0-4052-b3b7-0abd33424367.png)

When the log level is on INFO I can't get any useful information reading backend logs 

### What you expected to happen

I expect to have information on the whole period I select 

### How to reproduce

Start a skywalking server, ui version 8.8.1 with docker ( https://github.com/apache/skywalking/blob/master/docker ) and postgres as backend.

Open the UI -> Select a time range bigger than one hour 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
This is a little hard to tell. I can find this issue in demo env or  showcase. Have you checked the data in the database? 
I am going to move this to discussion until we know this is a bug.

**Reply by marco-simoncini** [reply]
yes, data it's still there. If i shift the one-hour time range the UI will retrieve the correct data. 

Obviosly if I make a select on db the data are retrieved correctly.

**Reply by wu-sheng** [reply]
One hour actually 60 minutes data, so it is different. There are hour-level data in the storage.

![image](https://user-images.githubusercontent.com/5441976/144942215-4eb22958-6fd0-4dfa-9169-5e4ba88e2d14.png)

This is on the demo.skywalking.apache.org, and one bar per hour is the graph expected. You could find the hour-level time bucket in the metric table.

**Reply by marco-simoncini** [reply]
Sorry but there's something I didn't understand. This's a screen from your demo. Bars are always at zero. Even if I select a half hour time range. Is it not working? 

![immagine](https://user-images.githubusercontent.com/70517698/145825257-9c8965b7-74bb-4678-a523-26b58a8dd5d1.png)

I'm having the same behaviour in my local environemnt, if I select a time range greater than an hour every bar becomes zero. 

I am new to skywalking can you show me where is the metric table? 

Thank you

**Reply by wu-sheng** [reply]
![image](https://user-images.githubusercontent.com/5441976/145827164-856e22a2-bfcd-4aeb-af1c-3c15178a32d0.png)

Nothing special. All things are there once you selected an instance of JVM with agent installed.

**Comment by marco-simoncini** [comment]
Ah ok thank you for the explaination. 

Can you give me some tips to investigate my problem? I still can't get bars for time range longer than one hour.

hour time range
![immagine](https://user-images.githubusercontent.com/70517698/145828014-c7eab027-9f01-4d4c-b8dc-305fdb716223.png)
Two hours time range 
![immagine](https://user-images.githubusercontent.com/70517698/145828172-f720dc33-ed68-45f0-b7d0-730a60861d87.png)

Probably there's something that i don't understand on how Skywalking works. Can you give me a hint? 

Thank you!

**Reply by wu-sheng** [reply]
Don't know what happens on private env. Sorry.
